### PR TITLE
Add Lavalink to recommendations

### PR DESCRIPTION
**Pull request by Frederikam**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [ ] Library interface (affecting end-user code) 
- [x] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

Adds Lavalink (and Lavalink-Client) as recommendations in the readme, as per Minn's request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -492,6 +492,19 @@ to understand a proper implementation.
...
+
+Lavalink is a popular standalone audio sending node based on Lavaplayer. Lavalink was built with scalability in mind,
+and allows streaming music via many servers. It supports most of Lavaplayer's features.
+
+Lavalink is used by many large bots, as well as bot developers who can not use a Java library like Lavaplayer.
+If you plan on serving music on a smaller scale with JDA it is often preferable to just use Lavaplayer directly
+as it is easier.
+
+[Lavalink-Client](https://github.com/FredBoat/Lavalink-Client) is the official Lavalink client for JDA.
+Both Lavalink-Client and Lavalink are created and maintained by [Frederik Mikkelsen](https://github.com/Frederikam),
```
Comment (kantenkugel) [inline README.md]: Would move that one to the top similar to the other entries:

> Created and maintained by [Frederik Mikkelsen](https://github.com/Frederikam)

File: README.md [hunk 2]
```diff
@@ -492,6 +492,19 @@ to understand a proper implementation.
...
+### [Lavalink](https://github.com/frederikam/Lavalink)
+
+Lavalink is a popular standalone audio sending node based on Lavaplayer. Lavalink was built with scalability in mind,
+and allows streaming music via many servers. It supports most of Lavaplayer's features.
+
+Lavalink is used by many large bots, as well as bot developers who can not use a Java library like Lavaplayer.
+If you plan on serving music on a smaller scale with JDA it is often preferable to just use Lavaplayer directly
+as it is easier.
+
+[Lavalink-Client](https://github.com/FredBoat/Lavalink-Client) is the official Lavalink client for JDA.
```
Comment (kantenkugel) [inline README.md]: Would also add the Server part, as Client on its own is not enough to actually use Lavalink

Comment (Frederikam) [reply README.md]: The link is in the header

Comment (kantenkugel) [reply README.md]: Ok, seems fair

**Pull request merged**